bindata/network/ovn-kubernetes/managed/multi-zone-interconnect/ovnkube-node.yaml

```diff
@@ -819,6 +819,8 @@ spec:
           name: var-lib-openvswitch
         - mountPath: /run/openvswitch/
           name: run-openvswitch
+        - mountPath: /var/log/ovnkube/
```

Comment:
Note: you are using etc-openvswitch which is mounted to a different host-path in IC and non-IC, I think that should be fine though.

Reply:
right. This is only used in ovn-ic cases, so I hope that will not be too confusing.